spec/classes/lightblue/application/migrator_spec.rb

```diff
@@ -40,9 +40,9 @@
         }
       )
       
-      should contain_file(client_config) \
-        .that_comes_before("Service[#{service_name}]") \
-        .that_notifies("Service[#{service_name}]")
+      should contain_lightblue__client__configure(client_config) \
```

Comment:
There is a few thinks that I don't understand here:
* what is `contain_lightblue__client__configure` and where is it defined?
* if it should notify a service, what is the "comes before" bit afterwards?